plugins/woocommerce/includes/admin/class-wc-admin-settings.php

```diff
@@ -492,6 +492,7 @@ class="<?php echo esc_attr( $value['class'] ); ?>"
 							?>
 								<tr valign="top" class="<?php echo esc_attr( implode( ' ', $visibility_class ) ); ?>">
 									<th scope="row" class="titledesc"><?php echo esc_html( $value['title'] ); ?></th>
+									<td class="help-tooltip"><?php echo isset( $value['tooltip'] ) && '' !== $value['tooltip'] ? wc_help_tip( esc_html( $value['tooltip'] ) ) : ''; ?></td>
```

Comment:
Interestingly, this new cell creates an issue on the **WooCommerce ▸ Settings ▸ Advanced ▸ Custom Data Stores** screen, effectively displacing some other settings (because the rows above and below contain just two table cells):

<img width="1275" alt="wc-settings-advanced-custom-data-stores" src="https://user-images.githubusercontent.com/3594411/213327098-814e569b-43dc-483d-9fe3-ba2ff6beb2e0.png">

I haven't had a chance to look in any depth—I just noticed while working on some HPOS stuff—but I wonder if this is causing a problem on other screens, too (and/or what the best solution is).

cc @octaedro

Reply:
Does indeed impact a few places:

<img width="1325" alt="settings-advanced" src="https://user-images.githubusercontent.com/3594411/213765844-115b8d75-9cc0-4f7a-80e6-8e63c4fce242.png">

(Settings > Advanced)

Reply:
Thank you @barryhughes for reporting this.
I just [created a PR](https://github.com/woocommerce/woocommerce/pull/36531) to fix that.